src/main/resources/swagger/api-docs.yaml

```diff
@@ -151,6 +161,42 @@ paths:
             - email
             - profile
 
+  /api/admin/user/{userId}/enable:
```

Comment:
there's also an admin endpoint for DELETE /api/admin/user/{userId} which deletes the user from LDAP, but i haven't documented it in swagger because it's strictly for dev/qa purposes